src/main/kotlin/com/ajaxproject/warehouse/controller/CustomersController.kt

```diff
@@ -19,26 +19,30 @@ import org.springframework.web.bind.annotation.RestController
 
 @RestController
 @RequestMapping("/customers")
-class CustomersController(val customerService: CustomerService) {
+class CustomersController(
+    val customerService: CustomerService
+) {
 
     @GetMapping
-    fun findAllCustomers(): List<CustomerDataLiteDto> = customerService.findAllCustomers()
+    fun findAll(): List<CustomerDataLiteDto> = customerService.findAllCustomers()
 
     @GetMapping("/{id}")
-    fun findById(@PathVariable id: Int): CustomerDataDto = customerService.findById(id)
+    fun findById(@PathVariable id: String): CustomerDataDto =
+        customerService.getById(id)
 
     @PostMapping("/create")
     @ResponseStatus(HttpStatus.CREATED)
-    fun createCustomer(@RequestBody @Valid createDto: CustomerCreateDto): CustomerDataDto =
-        customerService.createCustomer(createDto)
+    fun createCustomer(
+        @RequestBody @Valid createDto: CustomerCreateDto
+    ): CustomerDataDto = customerService.createCustomer(createDto)
 
     @PutMapping("/{id}")
     fun updateCustomer(
-        @RequestBody @Valid updateDto: CustomerUpdateDto,
-        @PathVariable id: Int
+        @PathVariable id: String,
+        @RequestBody @Valid updateDto: CustomerUpdateDto
```

Comment:
not sure that `@Valid` really works here.

Could you check it please? Just request the endpoint with invalid data

Reply:
It works, I test my endpoints with Postman